src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/QF-1912-UpdatePendingUsersToLive.sql

```diff
@@ -0,0 +1,12 @@
+﻿/*
+	Update User Status to Live for users with Pending Status and GovUkIdentifier is not null or empty
```

Comment:
Explain that this is a temporary script which when deployed to all environments after QF-1784 will correct the existing data and then this script can be removed.

Reply:
Updated description in update script file and post deployment script file